plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/StandardProjectWizard.java

```diff
@@ -20,12 +39,18 @@
 
 public class StandardProjectWizard extends Wizard implements INewWizard {
 
-  private AppEngineStandardWizardPage page;
+  private WizardPage page;
   private AppEngineStandardProjectConfig config = new AppEngineStandardProjectConfig();
+  private boolean hasAppEngineComponent;
 
   public StandardProjectWizard() {
     this.setWindowTitle("New App Engine Standard Project");
-    page = new AppEngineStandardWizardPage();
+    hasAppEngineComponent = doesAppEngineJavaComponentExist();
+    if (hasAppEngineComponent) {
```

Comment:
Would it make sense to:
  - prepend `AppEngineComponentPage` to be the first page, such that `AppEngineStandardWizardPage` will be the second page
  - have `AppEngineComponentPage` have a _Check_ button that would run the `doesAppEngineJavaComponentExist()` check
  - and `AppEngineComponentPage#canFlipToNextPage()` check that `doesAppEngineJavaComponentExist()` is true

Otherwise the user needs to cancel and restart the new wizard.